pkg/swap/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.5
+
+# swap command - only minimal Alpine install
+MAINTAINER Avi Deitcher <https://github.com/deitch>
```

Comment:
`MAINTAINER` is deprecated, and its not helpful in this context can you remove...